CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.13)
```

Comment:
Was that file generated, too? Or did you add it? Wasn't sure from the commit.

Reply:
Yes, this was generated from `tree-sitter init -u`.